src/org/etools/j1939_84/controllers/part11/Part11Step13Controller.java

```diff
@@ -644,108 +665,131 @@ private void testSp12675(OBDModuleInformation module) {
+            if (getEngineModelYear() >= 2022 && getEngineModelYear() <= 2023) {
+                addWarning("6.11.13.4.b - No response was received from "
+                        + module.getModuleName());
+            }
+        } else {
+            // 6.11.13.3.b - List data received in a table using bin numbers for rows.
+            getListener().onResult(nOxBinningModule.format(nOx100HourPackets));
+
+            nOx100HourPackets.stream().forEach(genericPacket -> {
+                var partTwoPacket = get(genericPacket.getPgnDefinition().getId(), module.getSourceAddress(), 2);
+
+                genericPacket.getSpns().forEach(spn -> {
+                    var checkValue = spn.getSlot().getByteLength() == 2 ? 0xFAFFL : 0xFAFFFFFFL;
+                    var checkValueString = spn.getSlot().getByteLength() == 2 ? String.format("0x%04X", checkValue)
+                            : String.format("0x%08X", checkValue);
                     if (spn.getRawValue() > checkValue) {
                         // 6.11.13.4.c. Fail each PG query where any bin value received is greater than FAFFh. (Use
                         // FAFFFFFFh for NOx values)
-                        addFailure("6.11.13.4.c - Bin value received is greater than " + checkValueString + "(h) from "
-                                + module.getModuleName() + " for " + spn);
+                        addFailure(
+                                   "6.11.13.4.c - Bin value received is greater than " + checkValueString + "(h) from "
+                                           + module.getModuleName() + " for " + spn);
 
                     }
                     // 6.11.13.4.d. Fail all values where the corresponding value received in part 2 is greater than the
                     // part 11 value (where supported)
-                    var partTwoValue = partTwoPacket.getSpnValue(spn.getId()).orElse(NOT_AVAILABLE);
-                    if (partTwoValue > spn.getValue()) {
-                        addFailure("6.11.13.4.d - Value received from " + module.getModuleName()
-                                           + " for " + spn
-                                           + " in part 2 was greater than part 11 value");
-                    }
+                    var partTwoValue = partTwoPacket.getSpnValue(spn.getId())
+                                                    .orElse(NOT_AVAILABLE);
 
-                    // 6.11.13.4.e.  Fail if active 100 hrs engine hours bin 1 SP 12389 < 600 seconds (where supported)
+                    // 6.11.13.4.e. Info if active 100 hrs engine hours bin 1 SP 12389 < 60 seconds (where supported)
                     if (spn.getId() == 12389) {
-                        if (spn.getValue() < 600) {
-                            addWarning("6.11.13.4.e - Active 100 hrs engine hours bin 1 SP " + spn.getId() + " value received is < 600 seconds from "
+                        if (spn.getValue() < 60) {
```

Comment:
This might need a check for `hasValue()` as the requirements indicated "where supported" which means some modules might send N/A?